packages/grid/_modules_/grid/components/column-header-item.tsx

```diff
@@ -90,9 +90,11 @@ export const ColumnHeaderItem = ({
       key={column.field}
       data-field={column.field}
       style={{
-        width,
-        minWidth: width,
-        maxWidth: width,
+        flex: column.flex == null ? `0 0 ${column.width}` : `${column.flex} ${column.flex} `,
+
```

Comment:
How about we do it this way?

Reply:
Won't this mess up the way the columns widths are calculated if it for instance you resize a column?

Reply:
Yeah I had a play, it will be difficult to manage, let's do with the width!